src/app/loadout-builder/processWorker/process.ts

```diff
@@ -352,39 +436,37 @@ function getStatValuesWithMWProcess(
 
   // Checking energy tells us if it is Armour 2.0 (it can have value 0)
   if (item.sockets && item.energy) {
-    let masterworkSocketHashes: number[] = [];
-    // only get masterwork sockets if we aren't manually adding the values
-    if (!assumeMasterwork) {
+    if (assumeMasterwork || item.energy) {
+      // TODO: technically we could derive this from the available mods instead ("slot" them all)
+      // Alternately we could make a lot more assumptions and just say if the energy capacity is 10, add 2 to every stat
```

Comment:
Yeah I have had this thought before but kept the more 'correct' way. I think now that the energy system seems here to stay making the assumption would be fine.